File: grpc-starter/grpc-starter-spring-boot2-examples/src/main/java/io/github/jojoti/grpcstatersbexamples/FooHandler.java
package io.github.jojoti.grpcstatersbexamples;

import io.github.jojoti.grpcstartersb.FooGrpc;
import io.github.jojoti.grpcstartersb.Hello;
import io.github.jojoti.grpcstatersb.GRpcPrivateService;
import io.grpc.stub.StreamObserver;

/**
 *
 * @author JoJo Wang
 * @link github.com/jojoti
 */
@GRpcPrivateService
public class FooHandler extends FooGrpc.FooImplBase {

    public FooHandler() {

    }

    @Override
    public void bar(Hello.BarRequest request, StreamObserver<Hello.BarResponse> responseObserver) {
        super.bar(request, responseObserver);
    }

}

File: grpc-starter/grpc-starter-spring-boot2-kotlin-examples/src/main/kotlin/io/github/jojoti/grpcstatersbexamples/FooHandler.kt
package io.github.jojoti.grpcstatersbexamples

import io.github.jojoti.grpcstatersb.GRpcPrivateService
import io.github.jojoti.grpcstatersbexamples.FooGrpcKt
import io.github.jojoti.grpcstatersbexamples.Hello
import kotlin.coroutines.CoroutineContext

/**
 *
 * @author JoJo Wang
 * @link github.com/jojoti
 */
@GRpcPrivateService
open class FooHandler : FooGrpcKt.FooCoroutineImplBase() {

    override suspend fun bar(request: Hello.BarRequest): Hello.BarResponse {
        return Hello.BarResponse.newBuilder().build();
    }
}